specification/azurearcdata/resource-manager/Microsoft.AzureArcData/preview/2023-11-01-preview/examples/CreateOrUpdateArcSqlServerAvailabilityGroup.json

```diff
@@ -0,0 +1,259 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "testrg",
+    "sqlServerInstanceName": "testSqlServer_INST1",
+    "availabilityGroupName": "testAG",
+    "api-version": "2023-11-01-preview",
+    "sqlServerAvailabilityGroupResource": {
+      "location": "southeastasia",
+      "tags": {
+        "mytag": "myval"
+      },
+      "properties": {
+        "availabilityGroupId": "00000000-1111-2222-3333-444444444444",
+        "serverName": "testSqlServer",
+        "instanceName": "INST1",
+        "collectionTimestamp": "2022-05-05T16:26:33.883Z",
+        "info": {
+          "failureConditionLevel": 3,
+          "healthCheckTimeout": 30000,
+          "automatedBackupPreferenceDescription": "SECONDARY",
+          "version": 0,
+          "basicFeatures": false,
+          "dtcSupport": false,
+          "dbFailover": true,
+          "isDistributed": false,
+          "clusterTypeDescription": "WSFC",
+          "requiredSynchronizedSecondariesToCommit": 0,
+          "isContained": false,
+          "primaryReplica": "testSqlServer\\INST1",
+          "primaryRecoveryHealthDescription": "ONLINE",
+          "secondaryRecoveryHealthDescription": null,
```

Comment:
You shouldn't have null in examples unless the property is modeled as 'x-ms-nullable' true.

Also, its recommended NOT to model properties as 'x-ms-nullable' true, as a general rule.